hack/make-rules/test-integration.sh

```diff
@@ -90,8 +90,17 @@ if [[ -n "${KUBE_TEST_ARGS}" ]]; then
   runTests v1
 fi
 
+# Pass arguments that begin with "-" and move them to goflags.
+what_flags=""
+for arg in "$@"; do
+  if [[ "${arg}" == -* ]]; then
+    what_flags="${what_flags} \"${arg}\""
+  fi
+done
+
+
 # Convert the CSV to an array of API versions to test
 IFS=';' read -a apiVersions <<< "${KUBE_TEST_API_VERSIONS}"
 for apiVersion in "${apiVersions[@]}"; do
-  runTests "${apiVersion}" "${1-}"
+  runTests "${apiVersion}" "${1-}" "${what_flags}"
```

Comment:
This won't work how you think it will. You're always going to have *one* argument to `runTests` that you break up later, but it will break like this:
```sh
myvar="\"-first\" \"-second=val\" \"-third otherval\""
...
runTests "${apiVersion}" "${1-}" "\"-first\" \"-second=val\" \"-third otherval\""
...
make
    ...
    WHAT="... \"-first\" \"-second=val\" \"-third otherval\""
              ^--arg0--^ ^----arg1-----^ ^-arg2-^ ^--arg3--^
```

What you want to do is:
 - make `$what_flags` into an array
 - append to the array correctly (`array+=( "new item with spaces or whatever" )`)
 - expand the array into args *here* `runTests ... "${what_flags[@]}"`
 - expect varargs in `runTests` from arg 3 onward `WHAT="$( whatever ) ${@:4}"